### PR TITLE
feat: :sparkles: Added projections from mandroc to pyro

### PR DESCRIPTION
**Pull request by Auxtal**
I have added the old projections from Mandroc to pyro to add some more interaction to the server. Removed old imports not being used. Has been tested in my testing guild.

### CODE REVIEW

File: cogs/menudocs.py [hunk 1]
```diff
@@ -97,6 +88,62 @@ async def on_thread_join(self, thread) -> None:
...
+        ]
+
+        projections = after.guild.get_channel(MENUDOCS_PROJECTIONS_CHANNEL)
+        unverified_role = after.guild.get_role(MENUDOCS_UNVERIFIED_ROLE)
+        message = Template(random.choice(welcome_messages)).safe_substitute(
+            {"MEMBER": f"**{after}**"}
+        )
+        embed = disnake.Embed(description=message, colour=disnake.Colour.green())
+
+        if unverified_role in before.roles:
```
Comment (Skelmis) [inline cogs/menudocs.py:119]: This also needs to check that the unverified role isn't in the after.roles, otherwise anything could trigger it

**Review by Skelmis — CHANGES_REQUESTED** [review]

File: cogs/menudocs.py [hunk 2]
```diff
@@ -97,6 +88,62 @@ async def on_thread_join(self, thread) -> None:
...
+            "Whoopsies! $MEMBER left us!",
+            "Nooooooo, $MEMBER closed the door.",
+            "Commander, we've lost $MEMBER!",
+            "Is this a loss? $MEMBER left."
+        ]
+
+        message = Template(random.choice(leave_messages)).safe_substitute(
+            {"MEMBER": f"**{member}**"}
+        )
+        projections = member.guild.get_channel(1003987467612991508)
```
Comment (Skelmis) [inline cogs/menudocs.py:142]: Is this also not the projections variable we already have?

Comment (Auxtal) [reply cogs/menudocs.py:142]: Oh yeah, that's my channel whoops.

**Review by Skelmis — APPROVED** [review]
lgtm

**Pull request merged**